sdk/src/main/java/com/silanis/esl/sdk/internal/converter/ReferencedFieldConverter.java

```diff
@@ -0,0 +1,55 @@
+            apiReferencedInCondition.add(converter.toAPIFieldCondition());
+        }
+
+        List<com.silanis.esl.api.model.FieldCondition> apiReferencedInAction = new ArrayList<com.silanis.esl.api.model.FieldCondition>();
+        for (com.silanis.esl.sdk.FieldCondition sdkFieldCondition : sdkReferencedFieldConditions.getReferencedInAction()) {
+            FieldConditionConverter converter = new FieldConditionConverter(sdkFieldCondition);
+            apiReferencedInAction.add(converter.toAPIFieldCondition());
+        }
+
+        com.silanis.esl.api.model.ReferencedFieldConditions apiReferencedFieldConditions = new com.silanis.esl.api.model.ReferencedFieldConditions();
+        apiReferencedFieldConditions.setReferencedInCondition(apiReferencedInCondition);
+        apiReferencedFieldConditions.setReferencedInAction(apiReferencedInAction);
+
+        com.silanis.esl.api.model.ReferencedField apiReferencedField = new com.silanis.esl.api.model.ReferencedField();
+        apiReferencedField.setFieldId(sdkReferencedField.getFieldId());
+        apiReferencedField.setConditions(apiReferencedFieldConditions);
+        return apiReferencedField;
+    }
+
+    public static com.silanis.esl.sdk.ReferencedField toSDK(com.silanis.esl.api.model.ReferencedField apiReferencedField) {
+        com.silanis.esl.api.model.ReferencedFieldConditions apiReferencedFieldConditions = apiReferencedField.getConditions();
+        List<com.silanis.esl.sdk.FieldCondition> sdkReferencedInCondition = new ArrayList<com.silanis.esl.sdk.FieldCondition>();
+        for (com.silanis.esl.api.model.FieldCondition apiFieldCondition : apiReferencedFieldConditions.getReferencedInCondition()) {
+            FieldConditionConverter converter = new FieldConditionConverter(apiFieldCondition);
+            sdkReferencedInCondition.add(converter.toSDKFieldCondition());
+        }
+
+        List<com.silanis.esl.sdk.FieldCondition> sdkReferencedInAction = new ArrayList<com.silanis.esl.sdk.FieldCondition>();
+        for (com.silanis.esl.api.model.FieldCondition apiFieldCondition : apiReferencedFieldConditions.getReferencedInAction()) {
+            FieldConditionConverter converter = new FieldConditionConverter(apiFieldCondition);
+            sdkReferencedInAction.add(converter.toSDKFieldCondition());
+        }
+
+        com.silanis.esl.sdk.ReferencedFieldConditions sdkReferencedFieldConditions = new com.silanis.esl.sdk.ReferencedFieldConditions();
+        sdkReferencedFieldConditions.setReferencedInCondition(sdkReferencedInCondition);
+        sdkReferencedFieldConditions.setReferencedInAction(sdkReferencedInAction);
+
+        com.silanis.esl.sdk.ReferencedField sdkReferencedField = new com.silanis.esl.sdk.ReferencedField();
+        sdkReferencedField.setFieldId(apiReferencedField.getFieldId());
+        sdkReferencedField.setConditions(sdkReferencedFieldConditions);
+        return sdkReferencedField;
+    }
+}
```

Comment:
When you create a new converter, you need to create a unit test for it.
Add ReferencedFieldConverterTest.

Reply:
done